test/test_jit.py

```diff
@@ -13126,10 +13126,10 @@ class TestJitGeneratedFunctional(JitTestCase):
     ('softmin', (S, S, S), (0,),),
     ('softmax', (S, S, S), (0,), '', (True,)),
     ('softmax', (S, S, S), (0, 3, torch.double), 'with_all_args', (True,)),
-    ('tanh', (S, S, S), (),),
-    ('sigmoid', (S, S, S), (),),
+    ('tanh', (S, S, S), (), '', (True,)),
+    ('sigmoid', (S, S, S), (), '', (True,)),
     ('log_softmax', (S, S, S), (0,), '', (True,)),
-    ('linear', (S, S), ((M, S),),),
+    ('linear', (S, S), ((M, S),), '', (True, ['aten::t', 'aten::matmul'])),
```

Comment:
This means we can't autodiff aten::t and aten::matmul right?

Reply:
This means we autodiff aten::t and aten::matmul and they must appear in the differential graph to pass the test :)

Reply:
I take it we can't autodiff addmm then ?

Reply:
addmm is getting decomposed to mm + add somewhere before autodiff, so we autodiff `mm + add` in this case.

Reply:
Yea actually this test only trigger the second part of `linear` impl here, I'm adding another test to trigger the `add + mm` path. Nice catch @eellison @wanchaol !

Reply:
reference link:https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1408